ci-operator/jobs/openshift/microshift/openshift-microshift-main-periodics.yaml

```diff
@@ -1021,6 +1105,62 @@ periodics:
     ci-operator.openshift.io/variant: periodics
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-microshift-main-periodics-rebase-on-nightlies
```

Comment:
Eventually we'll have 1 of these jobs for each supported branch, right? Should the version number be included in the name?

Reply:
The pattern seems to be to include the branch name into the job + job file name. So guess "main" is correct here and we'd add a "release-4.x" version once we create a corresponding branch.